WTWSMS/events/ACR_misc_hellenic_flavor.txt

```diff
@@ -5419,6 +5438,7 @@ character_event = {
 		NOT = { has_character_flag = VIET_artemis_sex_question }
 		NOT = { trait = celibate }
 		NOT = { trait = eunuch }
+		supernatural_events_trigger = yes
 	}
 	
 	weight_multiplier = {
```

Comment:
Once again: Artemis literally heals the player, and meets nonexistent, mythological beasts in supernatural adventures. Even someone who thinks they have spoken to a deity doesn't go speak to the deity with syphilis and come back without it.